XiEditor/EditView.swift

```diff
@@ -94,6 +94,7 @@ class EditView: NSView, NSTextInputClient {
             return NSColor(colorLiteralRed: 0.8, green: 0.8, blue: 0.8, alpha: 1.0)
         }
     }
+    var textHighlightColor: NSColor = NSColor(deviceWhite: 0.8, alpha: 0.4)
```

Comment:
I guess this will come from themes soon? (We will be able to do themes very soon)